### PR TITLE
Section table material

### LINKED ISSUE #155: Dark Mode Improvements
- [x] Schedule of classes improvements 
image
- [x] Tool tip link visibility 
image
- [x] Loading video

### PR DESCRIPTION
**Pull request by devsdevsdevs**
## Summary
Implemented Material-UI table for compatible (and improved) aesthetics and compatibility with Dark Mode.

image

image

## Test Plan
- Confirm alignment of all tables' columns
- Test table appears fine on different size screens/DPIs
- Test table components are interact well on both light/dark mode (popups/buttons/hovers etc.)

## Issues
#155 #133 

## Future Followup
Component optimizations: we can make mounting of some components more efficient. Not all tables need to listen to events from the store individually etc.

### CODE REVIEW

**Comment by devsdevsdevs** [comment]
Need to add OS dark mode detection and auto switching before merge

File: client/src/components/SectionTable/SectionTableBody.js [hunk 1]
```diff
@@ -73,9 +70,6 @@ const styles = (theme) => ({
     full: {
         color: '#e53935',
     },
-    multiline: {
-        whiteSpace: 'pre',
-    },
```
Comment (ChaseC99) [inline client/src/components/SectionTable/SectionTableBody.js:78]: I think this is where the merge conflict is. Just make sure to keep my cursor change when you resolve it 👍

File: client/src/components/SectionTable/SectionTable.js [hunk 2]
```diff
@@ -79,34 +97,73 @@ class SectionTable extends PureComponent {
...
+                <TableContainer
+                    component={Paper}
+                    style={{ margin: '8px 0px 8px 0px' }}
+                    elevation={0}
+                    variant="outlined"
+                >
+                    <Table className={classes.table} size="small">
+                        <TableHead>
+                            <TableRow>
+                                <TableCell classes={{ sizeSmall: classes.cellPadding }} className={classes.row} />
```
Comment (ChaseC99) [inline client/src/components/SectionTable/SectionTable.js:131]: It seems like every TableCell has `classes={{ sizeSmall: classes.cellPadding }}` added to it.
Would it make sense to add `sizeSmall: classes.cellPadding` to `classes.row` and `classes.cell` (or, if possible, directly to TableCell), so we don't have to add the other classes statement everywhere?

**Review by ChaseC99 — APPROVED** [review]
I'm so excited to use dark mode, this looks great!!

**Comment by ChaseC99** [comment]
Will this also close #133? Or is there more for that one?

**Pull request merged**